README.md

```diff
@@ -4,6 +4,10 @@ An OPTIMADE Client as a Materials Cloud tool.
 
 It runs a Jupyter notebook in Voilà with the [Materials Cloud Voilà template](https://github.com/materialscloud-org/voila-materialscloud-template).
 
+The test deployment is on https://optimadeclient.matcloud.xyz/, the production deployment is on https://www.materialscloud.org/work/tools/optimadeclient. 
```

Comment:
Here I would use either the direct dokku url for both (preferred imo), or the materials cloud url, where the dokku page is embedded via iframes.

So:

https://optimadeclient.matcloud.xyz/ and https://optimadeclient.materialscloud.io/

or

https://dev-www.materialscloud.org/work/tools/optimadeclient and https://www.materialscloud.org/work/tools/optimadeclient

Reply:
Agree! thanks.